src/content/basics/nodepools/index.md

```diff
@@ -139,6 +139,14 @@ Just as the Giant Swarm API schema for v4 (without node pools) and v5 (with node
 The new definition schema for v5 allows for defining cluster and node pool details in one file,
 to be submitted for creation via the [`gsctl create cluster`](/reference/gsctl/create-cluster/) command.
 
+## Node pools and autoscaling {#autoscaling}
+
+With node pools, you set the autoscaling range per node pool. The Kubernetes cluster autoscaler has to decide which node pool to scale under which circumstances.
```

Comment:
```suggestion
With node pools, the autoscaling is configured per node pool automatically. The Kubernetes cluster autoscaler has to decide which node pool to scale under which circumstances.
```
In reality customer does not perform any action

Reply:
Actually, yes. The customer initially decides in which range the node pool can scale.

Reply:
in which range? can you explain it a bit more? what do you mean by range?

Reply:
I mean the number of nodes, which is configured by the user as a MIN and MAX number.

Reply:
AH ok I was thinking from the ASG configuration side